neo/Persistence/Prefixes.cs

```diff
@@ -12,9 +12,7 @@ internal static class Prefixes
         public const byte IX_CurrentBlock = 0xc0;
         public const byte IX_CurrentHeader = 0xc1;
 
-        public const byte SYS_Version = 0xf0;
```

Comment:
Hello, any reason to remove this? Don't all implementations need the sys_version?

Reply:
`MemoryStore` doesn't need it.